content/arkly.ttl

```diff
@@ -3,5 +3,6 @@
 :arkly
     skos:prefLabel "arkly"@en;
     skos:definition '''
-A decentralized digital archive and permanent storage solution.
+      A next-generation decentralized digital archives
```

Comment:
We've been through the problems of next generation once with Orcfax once being a next generation Oracle until it couldn't be defined, or was no longer one. I also feel we should avoid marketing speak in the glossary even if it remains in other documentation, the glossary should be matter of fact and disambiguated, yet here, next generation poses lots of questions for integrators and archives people alike,